sor/src/main/java/com/bazaarvoice/emodb/sor/core/DistributedCompactor.java

```diff
@@ -52,11 +53,11 @@ public Expanded expand(Record record, long fullConsistencyTimestamp, long compac
      */
     protected Expanded doExpand(Record record, long fullConsistencyTimestamp, long compactionConsistencyTimeStamp, long compactionControlTimestamp, MutableIntrinsics intrinsics, boolean ignoreRecent)
             throws RestartException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'com.bazaarvoice.emodb.sor.core.AbstractCompactor$RestartException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.qa.bazaarvoice.com/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)